docs/Developers/Codestyle.rst

```diff
@@ -61,8 +61,8 @@ with the allowed types:
             c = ""
         return a + b + c
 
-This will raise a ``TypeError`` if ``a``, ``b`` or ``c`` are not strings
-and ``c`` is not ``None``.
+This will raise a ``TypeError`` if ``a`` or ``b`` are not strings or ``c`` is not a string
```

Comment:
@sravi4701 Before pushing, always run coala in your cloned directory.
It will find out the errors and report / fix them.
For now,
Make the changes, `git add` them and then run `git commit --amend`.

Also note that, along with the above error, your commit message is also not proper (although its better than many newcomers)
Please go through coala.io/commit again and dont forget to add the issue url at the end.

After you have made the changes in the file and the commit message you need to `git push` again. But you will need to use `git push --force`, to overwrite the above commit.